UnityProject/Assets/Scripts/Systems/Spells/Spell.cs

```diff
@@ -248,6 +250,9 @@ public virtual bool ValidateCast(PlayerInfo caster)
 
 		private bool CheckWizardGarb(Equipment casterEquipment)
 		{
+			// none-wizards shouldn't need a check for robes.
+			if (casterEquipment.ItemStorage.registerPlayer.PlayerScript.mind.GetAntag().Antagonist !=
```

Comment:
can GetAntag() return null?

Reply:
it can. You could add null propagation to the statement or us ``Mind.IsAntag`` before trying to access the antag. It is also weird how we get it with a method like this was old school java.

For these cases I think the better solution is using the ``Try`` pattern and return bools + out the wanted value if found.